src/hotspot/share/code/nmethod.hpp

```diff
@@ -25,6 +25,8 @@
 #ifndef SHARE_CODE_NMETHOD_HPP
 #define SHARE_CODE_NMETHOD_HPP
 
+#define IMMUTABLE_DATA_REFERENCES   int
```

Comment:
From https://github.com/openjdk/jdk/blob/master/doc/hotspot-style.md#naming
> Type names and global names should use mixed-case with the first letter of each word capitalized (FooBar).

Instead of macro, use `using AaaBbbb = int;`.
You can move it inside `nmethod` declaration.